_src/_includes/utility/testimonials.html

```diff
@@ -10,7 +10,15 @@
           {%- assign img = b.image -%}
           <!-- Text -->
           <p class="lead text-gray-700 mb-5 mb-md-7">
-            {{b.text1}}
+            {% assign textlimit = 295 %}
+            {% assign toggleclass = 'collapse' | append: forloop.index %}
+            {% if b.text1.size > textlimit %}
+            <span>{{ b.text1 | slice: 0, textlimit }}</span>
+            <span class="{{ toggleclass }} show">&hellip; <a class="text-muted text-nowrap" data-toggle="collapse" href=".{{ toggleclass }}" role="button" aria-expanded="false" aria-controls="{{ toggleclass }}">Read more &rarr;</a></span>
```

Comment:
```suggestion
            <span class="{{ toggleclass }} show">&hellip; <a class="text-muted text-nowrap" data-toggle="collapse" href=".{{ toggleclass }}" role="button" aria-expanded="false" aria-controls="{{ toggleclass }}">Read more</a></span>
```

I'd like to not have the arrow.